reactFrontenddd/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "reactFrontend",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^6.4.2",
```

Comment:
Let's pin all our libraries to avoid breaking changes in the future. Look up breaking changes if you don't fully understand this.